frontend/src/pages/modelServing/screens/projects/ModelServingPlatformSelect.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {
+  Alert,
+  AlertActionCloseButton,
+  Gallery,
+  GalleryItem,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import { ServingRuntimePlatform } from '~/types';
+import ModelServingPlatformCard from '~/pages/modelServing/screens/projects/ModelServingPlatformCard';
+
+type ModelServingPlatformSelectProps = {
+  onSelect: (platform: ServingRuntimePlatform) => void;
+  emptyTemplates: boolean;
+};
+
+const ModelServingPlatformSelect: React.FC<ModelServingPlatformSelectProps> = ({
+  onSelect,
+  emptyTemplates,
+}) => {
+  const [alertShown, setAlertShown] = React.useState(true);
+  return (
+    <Stack hasGutter>
+      <StackItem>
+        Select the type model serving platform to be used when deploying models in this project.
+      </StackItem>
+      <StackItem>
+        <Gallery hasGutter maxWidths={{ default: '400px' }}>
+          <GalleryItem>
+            <ModelServingPlatformCard
+              id="single-serving-platform-card"
+              onClick={() => onSelect(ServingRuntimePlatform.SINGLE)}
+              title="Single model serving platform"
+              description="Each model is deployed from its own model server. Choose this option when you have a small number of large models to deploy."
+              isModelMesh={false}
```

Comment:
I realize that this is done in this way because the button is reused in multiple places and encapsulates the two different actions. But I find it odd that the card tries to be generic as well as specific at the same time. You pass different titles and descriptions but also accept a flag `isModelMesh` in order to conditionally render different buttons later on.

Maybe accept the action as a prop?